README.md

```diff
@@ -48,18 +48,14 @@ Samples using the Steeltoe Security packages for Authentication and Authorizatio
 | --- | --- | --- |
 | [RedisDataProtection](Security/src/RedisDataProtection) | | [![Build Status](https://dev.azure.com/SteeltoeOSS/Steeltoe/_apis/build/status%2FSamples%2FSteeltoe-Samples-Security-RedisDataProtection?branchName=latest)](https://dev.azure.com/SteeltoeOSS/Steeltoe/_build/latest?definitionId=74&branchName=latest) |
 
-## [FreddysBBQ](FreddysBBQ)
-
-A polyglot (Java and .NET) microservices-based sample demonstrating interoperability between Java and .NET on Cloud Foundry, secured with OAuth2 Security Services and using Spring Cloud Services.
-
-# Branches
+## Branches
 
 All new development is done on the latest branch. Samples for the stable version of Steeltoe can be found on the main branch.
 
-# Documentation
+## Documentation
```

Comment:
These heading changes don't look right. At this level, the individual samples are listed. This is not one of them.

Reply:
I agree it's not ideal, but it's also not ideal to have 5 h1's on the page. I'm not sure I want to bother figuring out a better structure right now though, so I'll just revert all the header-level changes